src/Ursa/Controls/Dialog/OverlayDialog.cs

```diff
@@ -116,7 +130,15 @@ public static Task<DialogResult> ShowModal<TView, TViewModel>(TViewModel vm, str
         host.AddModalDialog(t);
         return t.ShowAsync<DialogResult>(token);
     }
-    
+
+    public static Task<DialogResult> ShowModal<TView, TViewModel>(string? hostId = null,
+    OverlayDialogOptions? options = null, CancellationToken? token = default)
+        where TView : Control, new()
+        where TViewModel : new()
+    {
+        return ShowModal<TView, TViewModel>(hostId, options, token);
```

Comment:
This is an infinite recursive call.

Reply:
I'm so sorry about such a stupid mistake